fdbserver/TLogServer.actor.cpp

```diff
@@ -1530,11 +1531,15 @@ void commitMessages(TLogData* self,
 					txsBytes += tagData->versionMessages.back().second.expectedSize();
 				}
 				if (SERVER_KNOBS->ENABLE_VERSION_VECTOR) {
-					auto iter = logData->waitingTags.find(tag);
-					if (iter != logData->waitingTags.end()) {
-						auto promise = iter->second;
-						logData->waitingTags.erase(iter);
-						promise.send(Void());
+					if (tags.present()) {
+						tags.get().push_back(tag);
```

Comment:
Seems the same tag can be inserted multiple times, which is probably not desirable?